frontends/concrete-python/concrete/fhe/tfhers/bridge.py

```diff
@@ -170,38 +172,92 @@ def serialize_input_secret_key(self, input_idx: int) -> bytes:
             bytes: serialized key
         """
         keyid = self._input_keyid(input_idx)
-        input_type = self._input_type(input_idx)
-        if input_type is None:  # pragma: no cover
-            msg = "input at 'input_idx' is not a TFHErs value"
-            raise ValueError(msg)
-        glwe_dim = input_type.params.glwe_dimension
-        poly_size = input_type.params.polynomial_size
         # pylint: disable=protected-access
-        return self.circuit.client.keys._keyset.serialize_lwe_secret_key_as_glwe(  # type: ignore
-            keyid, glwe_dim, poly_size
-        )
+        secret_key = self.circuit.client.keys._keyset.get_lwe_secret_key(keyid)  # type: ignore
         # pylint: enable=protected-access
+        return secret_key.serialize()
+
+    def keygen_with_initial_keys(
+        self,
+        input_idx_to_key_buffer: Dict[int, bytes],
```

Comment:
?

Reply:
deserialization requires knowing the parameters. So we have different options for the API:
- we can provide an API for deserializng keys in the bridge, then they will be passed in here
- keys should already come from TFHE-rs as bytes, so we are asking them to forward them here, and we will do the deserialization in the bridge (that's what I implemented)